### PR TITLE
use prx-datepicker for utc support

### PR DESCRIPTION
**Pull request by sandikbarr**
- uses the prx-datepicker for UTC support

### CODE REVIEW

File: src/app/campaign/campaign.component.ts [hunk 1]
```diff
@@ -102,10 +102,11 @@ export class CampaignComponent implements OnInit, OnDestroy {
   createFlight() {
     this.campaignStoreService.campaignFirst$.subscribe(state => {
       const flightId = Date.now();
+      const date = new Date(flightId);
       const flight = {
         name: 'New Flight ' + (Object.keys(state.flights).length + 1),
-        startAt: new Date().toISOString(),
-        endAt: new Date().toISOString(),
```
Comment (sandikbarr) [inline src/app/campaign/campaign.component.ts:108]: flight start date === flight end date is invalid, so I put in more sensible defaults for new flights, today thru the 1st of next month, which includes allocation dates thru the end of the month but not the 1st

File: src/app/campaign/flight/flight.component.html [hunk 2]
```diff
@@ -5,18 +5,8 @@
...
-        <input matInput [matDatepicker]="startPicker" placeholder="Actual Start date" formControlName="startAt" />
-        <mat-datepicker-toggle matSuffix [for]="startPicker"></mat-datepicker-toggle>
-        <mat-datepicker #startPicker></mat-datepicker>
-      </mat-form-field>
-      <mat-form-field appearance="outline">
-        <mat-label>Actual End Date</mat-label>
-        <input matInput [matDatepicker]="endPicker" placeholder="Actual End date" formControlName="endAt" />
-        <mat-datepicker-toggle matSuffix [for]="endPicker"></mat-datepicker-toggle>
-        <mat-datepicker #endPicker></mat-datepicker>
-      </mat-form-field>
```
Comment (sandikbarr) [inline src/app/campaign/flight/flight.component.html:19]: after many attempts with material datepicker, using ye ol prx-datepicker instead to get UTC support. (there is still a material date range picker in the backlog and a WIP.)

File: src/app/campaign/campaign.component.ts (hunk 1 above)
Comment (cavis) [reply src/app/campaign/campaign.component.ts:108]: +1

**Review by cavis — APPROVED** [review]
👍 works good!

**Pull request merged**